cdap-watchdog/src/main/java/co/cask/cdap/logging/save/LogSaver.java

```diff
@@ -135,17 +141,19 @@ private void createProcessors(Set<Integer> partitions) throws Exception {
     }
   }
 
-  private void cancelLogCollectorCallbacks() {
+  private Set<Cancellable> cancelLogCollectorCallbacks() {
```

Comment:
It stops the KafkaMessageCallbacks and then returns cancellables to cancel the Kafka partition consumers. I'll rename it to `stopKafkaMessageCallbacks()`.